src/cargo/core/profiles.rs

```diff
@@ -355,12 +382,13 @@ impl Profiles {
 /// An object used for handling the profile hierarchy.
 ///
 /// The precedence of profiles are (first one wins):
+///
 /// - Profiles in `.cargo/config` files (using same order as below).
-/// - [profile.dev.package.name] -- a named package.
-/// - [profile.dev.package."*"] -- this cannot apply to workspace members.
-/// - [profile.dev.build-override] -- this can only apply to `build.rs` scripts
+/// - `[profile.dev.package.name]` -- a named package.
```

Comment:
Aren't the backticks supposed to be within the link?

Reply:
Aha! I follow the cargo book 😅 

https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#overrides